docs/pvcscale.md

```diff
@@ -0,0 +1,197 @@
+# PVCScale Workload
+
+The PVCScale workload playbook is `workloads/pvcscale.yml` and will run the pvcscale workload on your cluster.
+
+PVCScale workload is designed to stress storage backend and provide information
+how fast PODS are stared depending on storage backend they require PVCs to be
+allocated from and attached to them.
+
+PVCScale test creates a desired number of PODS and attaches one PVC to each POD, it supports 
+only dynamic PVC allocation using storage class and in order for this test to 
+work it is requirement to have working storage class which can be used to allocate PVC.
```

Comment:
I think POD can just be referred to as pod and pvc as pvc or we should keep it consistent throughout all docs if we refer to pod as POD etc... (I'll have to check other docs if we do otherwise.)

Reply:
yup, fixed,